lib/Controller/DraftsController.php

```diff
@@ -175,9 +175,8 @@ public function update(int $id,
 		$message = $this->service->getMessage($id, $this->userId);
 		$account = $this->accountService->find($this->userId, $accountId);
 
-		($sendAt !== null)
-			? $message->setType(LocalMessage::TYPE_OUTGOING)
-			: $message->setType(LocalMessage::TYPE_DRAFT);
+	
+		$message->setType(LocalMessage::TYPE_DRAFT);
```

Comment:
`\OCA\Mail\Controller\DraftsController::create` still has the same logic and it is in use

1) Compose a new message
2) Send it before the first draft

There is a POST to create the *draft* but it results in implicit conversion to an outbox message.

Expectation is that the draft is created, then converted to an outbox message.